docs/readthedocs/source/doc/Nano/QuickStart/pytorch_train.md

```diff
@@ -40,13 +40,19 @@ trainer.fit(lightning_module, train_loader)
 
 #### Intel® Extension for PyTorch
 
-Intel Extension for Pytorch (a.k.a. IPEX) extends PyTorch with optimizations for an extra performance boost on Intel hardware. BigDL-Nano integrates IPEX through the `Trainer`. Users can turn on IPEX by setting `use_ipex=True`.
+Intel Extension for Pytorch (a.k.a. IPEX) [link](https://github.com/intel/intel-extension-for-pytorch) extends PyTorch with optimizations for an extra performance boost on Intel hardware. BigDL-Nano integrates IPEX through the `Trainer`. Users can turn on IPEX by setting `use_ipex=True`.
 
 ```python
 from bigdl.nano.pytorch import Trainer
 trainer = Trainer(max_epoch=10, use_ipex=True)
 ```
 
+Note: BigDL-Nano does not install IPEX by default. You can install IPEX using the following command:
```

Comment:
why not?

Reply:
how about ipex 1.11?

Reply:
Yes, IPEX 1.11 can be added to `setup.py`

Reply:
OK - then we need to understand what the performance will be without ipex (say, in ray tune) @MeouSker77

Reply:
> OK - then we need to understand what the performance will be without ipex (say, in ray tune) @MeouSker77

Ok, I'll do it.